test/functions.test.js

```diff
@@ -1,130 +1,488 @@
-    const expected = [
-        'Hello Suzi Summerson!',
-        'Hello Boot Penton!',
-        'Hello Cacilia Caramuscia!',
-        'Hello Cristal Oakman!',
-        'Hello Kenny Kepe!',
-        'Hello Dimitri Ochiltree!',
-        "Hello Ilaire O'Codihie!",
-        'Hello Anatol Gulley!',
-        'Hello Salomone Merkle!',
-        'Hello Alta Sulter!',
-        'Hello Alexandros Cornell!',
-        'Hello Thorn McMeekin!',
-        'Hello Esmaria Moukes!',
-        'Hello Munroe Walthall!',
-        'Hello Tannie Sedman!'
-    ];
-
-    const actual = greetUsersOverAge60(customers);
-
-    expect.deepEqual(actual, expected);
+test("greetUsersOverAge60", (expect) => {
+  const expected = [
+    "Hello Suzi Summerson!",
+    "Hello Boot Penton!",
+    "Hello Cacilia Caramuscia!",
+    "Hello Cristal Oakman!",
+    "Hello Kenny Kepe!",
+    "Hello Dimitri Ochiltree!",
+    "Hello Ilaire O'Codihie!",
+    "Hello Anatol Gulley!",
+    "Hello Salomone Merkle!",
+    "Hello Alta Sulter!",
+    "Hello Alexandros Cornell!",
+    "Hello Thorn McMeekin!",
+    "Hello Esmaria Moukes!",
+    "Hello Munroe Walthall!",
+    "Hello Tannie Sedman!",
+  ];
+
+  const actual = greetUsersOverAge60(customers);
+
+  expect.deepEqual(actual, expected);
 });
 
-skip('addAllAges', (expect) => {
-    const expected = true;
+test("addAllAges", (expect) => {
+  const expected = 2125;
 
-    const actual = true;
+  const actual = addAllAges(customers);
 
-    expect.equal(actual, expected);
+  expect.deepEqual(actual, expected);
 });
 
+test("getAverageCoolFactor", (expect) => {
+  const expected = 5.088888888888889;
 
-skip('getAverageCoolFactor', (expect) => {
-    const expected = true;
-
-    const actual = true;
+  const actual = getAverageCoolFactor(customers);
 
-    expect.equal(actual, expected);
+  expect.deepEqual(actual, expected);
 });
 
-skip('getTotalOfEachGender', (expect) => {
-    const expected = true;
+test("getTotalOfEachGender", (expect) => {
+  const expected = {
+    Bigender: 1,
+    Female: 19,
+    Genderqueer: 1,
+    Male: 23,
+    "Non-binary": 1,
+  };
 
-    const actual = true;
+  const actual = getTotalOfEachGender(customers);
 
-    expect.equal(actual, expected);
+  expect.deepEqual(actual, expected);
 });
 
-skip('getGenderBreakdownOfFordOwners', (expect) => {
-    const expected = true;
+test("getGenderBreakdownOfFordOwners", (expect) => {
+  const expected = {
+    Female: 1,
+    Male: 2,
+  };
 
-    const actual = true;
+  const actual = getGenderBreakdownOfFordOwners(customers);
 
-    expect.equal(actual, expected);
+  expect.deepEqual(actual, expected);
 });
 
-skip('getGenderBreakdownOfEachCar', (expect) => {
-    const expected = true;
-
-    const actual = true;
-
-    expect.equal(actual, expected);
+test("getGenderBreakdownOfEachCar", (expect) => {
+  const expected = {
+    "Alfa Romeo": {
+      Bigender: 1,
+      Female: 19,
+      Genderqueer: 1,
+      Male: 23,
+      "Non-binary": 1,
+    },
```

Comment:
Really close on this one. See how the gender break down is the same for each vehicle and also the same as your `getTotalOfEachGender` result? Your inside reduce just does the same reduction each time, a gender breakdown of all customers. if you could feed that inside reduce all customers for a given car type i think it would work perfectly